autoload/emoji_complete.vim

```diff
@@ -38,15 +38,23 @@ func! emoji_complete#complete()
     return ""
   endif
   call s:init_emojis_once()
-  let line = getline('.')[ : col('.')-1]
-  let s:complete_start = match(line, '\w*$')
+  " The text from the beginning of the line up until before the cursor.
+  let line = getline('.')[ : col('.')-2]
+  let s:complete_start = match(line, '\(-1\|+1\|[-a-zA-Z0-9_]*\)$')
   if s:complete_start >= 0
     augroup emoji_complete_done
       au!
       au CompleteDone <buffer> call s:emoji_expand()
     augroup end
-    call complete( s:complete_start+1, s:emojis )
-    return "\<c-p>"
+    let prefix = line[s:complete_start : ]
+    let matches = []
+    for emoji in s:emojis
+      if emoji['word'] =~? '^' . prefix
```

Comment:
Hi,

Why do you force matching from the beginning of the line? It could be interesting to do a matching on any part of the alias, allowing a word like `left` to present all the emoji which point to the left.

Kind,
Alexis